SPEC.md

```diff
@@ -547,6 +566,55 @@ This facilitates "wildcard" matching and a variety of common usage patterns, lik
 For example, an AppImage containing a set of bash scripts might omit both "os" and "arch", and hence could be used as a dependency by a variety of different AppImages.
 Alternatively, an AppImage might specify a dependency with no image ID and no "version" label, and the image discovery mechanism could always retrieve the latest version of an AppImage
 
+### Pod Manifest Schema
+
+JSON Schema for the Pod Manifest
+
+```
+{
+
+    "acVersion": "0.2.0",
+    "acKind": "PodManifest",
+    "apps": [
+        {
+            "app": "example.com/reduce-worker",
+            "imageID": "sha512-..."
```

Comment:
Erm, this is just because used bad examples in this manifest (will fix). Read the description below: the idea is to go with the discovery/dependency matching if people desire, but also to provide the option of referencing a canonical image. Hence the bidomality is intentional, otherwise pods could never be deterministic!

> there's no way to resolve an image based on its imageID...it implies images are already cached.

It suggests caching but no locality; I think it is perfectly feasible to anticipate many environments having non-local content addressable stores from which they would retrieve images based on hashes alone (ultimately I would love for there to be a global store for this a la [IPFS](http://ipfs.io/))